org.opent2t.sample.binaryswitch.superpopular/com.smartthings.binaryswitch/js/thingTranslator.js

```diff
@@ -98,6 +101,8 @@ function resourceSchemaToProviderSchema(resourceId, resourceSchema) {
 
     if ('power' === resourceId) {
         result['switch'] = resourceSchema.value ? 'on' : 'off';
+    }else{
+        throw new OpenT2TError(501, OpenT2TConstants.NotImplemented);
```

Comment:
Fixed.

---
In reply to: [106229721](https://github.com/openT2T/translators/pull/197#discussion_r106229721) [](ancestors = 106229721)